### ISSUE TITLE
Design Evaluation Survey

### ISSUE THREAD

**Issue by sharonmleon** [omeka/TrainingMaterials#5]
We need to be checking in with partipants about their satisfaction with the training.

**Comment by georgiawebgurl** [comment]
Yes, I have been thinking about this and had on my list to discuss. Have you used a standard form in the past? I copied one of my short feedback forms (Google form) and added an Omeka logo.  I also updated the verbiage of a couple of questions and added a question about the parts of training (as mentioned below). This might help inform the mix of training although I do think it may vary my institution. 
[Training Survey](https://forms.gle/viV76v6Sc6BRwLGL9) 

I added you as an editor.

**Comment by georgiawebgurl** [comment]
A made a copy of my short form & added an Omeka logo. 
[Training Survey](https://forms.gle/viV76v6Sc6BRwLGL9) 

I added you as an editor.

**Comment by georgiawebgurl** [comment]
I crafted a question about the format of training - if hands-on training, was there a good balance of exercises, demo, & content? Megan and I talked about this a bit. I think this may also be something to clarify in the training scope & agenda, too. I've tried to stress that students/trainees should be ready to do some work -- at least after the introductory session.

**Comment by sharonmleon** [comment]
We haven't really done these sorts of training sessions in a concerted way before, so we need to develop all this infrastructure from scratch.

**Comment by georgiawebgurl** [comment]
Ok, that is good to know and will help frame what my thoughts.

**Comment by sharonmleon** [comment]
I started to make some updates to this survey, but the more I look at it the more I think that we need specific ones for specific sessions. The "Did this meet the stated objectives" question is fairly unhelpful outside of the context of the individual client because it implies that there is a custom answer for each session. Really, there should be a core set of sessions that have articulated objectives and we should inquire about meeting each of them. Then, in the long run, data collected over a significant period of time will have some meaning. So, I guess this sends us back to the curriculum planning process.....
